include/torch-mlir/Dialect/Torch/IR/TorchOps.td

```diff
@@ -1158,6 +1158,24 @@ def Torch_ValsemVariantAtenFillScalarOp: Torch_Op<"valsem.aten.fill.Scalar", [
   let assemblyFormat = "$self `,` $value attr-dict `:` qualified(type($self)) `,` qualified(type($value)) `->` qualified(type($result))";
 }
 
+// The corresponding without underscore variant for `torch.aten.fill_.Tensor`
```

Comment:
This op does have an underscore variant. See: https://github.com/llvm/torch-mlir/pull/1531.